docs/dev/Pagination-v2.md

```diff
@@ -7,15 +7,15 @@ A cursor is a SQL abstraction for pagination. A client can open a cursor, retrie
 Currently, SQL plugin does not provide SQL cursor syntax. However, the SQL REST endpoint can return result a page at a time. This feature is used by JDBC and ODBC drivers.
 
 # Scope
-Currenty, V2 engine supports pagination only for simple `SELECT * FROM <table>` queries without any other clauses like `WHERE` or `ORDER BY`.
+At this time, V2 engine supports pagination only for simple `SELECT * FROM <table>` queries without any other clauses like `WHERE` or `ORDER BY`.
```

Comment:
I feel like we're going to have update this line of the design each time we push a PR.  
Maybe describe what the scope of this document contains.  They will have to look at release notes and read issues to know what has actually been pushed to repo.